### PR TITLE
LON-1023

### PR DESCRIPTION
**Pull request by SandraPG85**
Fix contextual menus from tree view and timeline

### CODE REVIEW

File: VAS.Drawing/Widgets/PlaysTimeline.cs [hunk 1]
```diff
@@ -285,18 +285,9 @@ protected virtual void RemoveEventTypeTimeline (EventTypeTimelineVM timelineVM)
 
 		protected void ShowTimersMenu (Point coords)
 		{
-			if (PeriodsTimeline != null &&
```
Comment (ylatuya) [inline VAS.Drawing/Widgets/PlaysTimeline.cs]: This a bit strange... the second path wasn't used in the old version?

File: VAS.UI.Gtk2/UI/Menu/MenuHelpers.cs [hunk 2]
```diff
@@ -83,9 +83,13 @@ static public void FillAddToPlaylistMenu (MenuItem addToPlaylistMenu, IEnumerabl
 				};
 			}
 			item = new MenuItem (Catalog.GetString ("Create new playlist..."));
-			LicenseLimitationVM limitation = App.Current.LicenseLimitationsService.Get ("Presentations");
-			if (limitation != null) {
-				item.Sensitive = limitation.Count < limitation.Maximum;
+
+			// FIXME: Longomatch has not implemented the limitation service, remove the null check when it is done
```
Comment (ylatuya) [inline VAS.UI.Gtk2/UI/Menu/MenuHelpers.cs]: Maybe fixed in https://github.com/fluendo/VAS/pull/221 already ?

File: VAS.Drawing/Widgets/PlaysTimeline.cs (hunk 1 above)
Comment (SandraPG85) [inline VAS.Drawing/Widgets/PlaysTimeline.cs]: In the previous version the periodstimelineview do not exist. This has code has been extracted and moved to the LMPlaysTimeline

File: VAS.UI.Gtk2/UI/Menu/MenuHelpers.cs (hunk 2 above)
Comment (SandraPG85) [inline VAS.UI.Gtk2/UI/Menu/MenuHelpers.cs]: no, but it will be fixed in a future user story I added the check to allow menus appear in the meantime.

**Pull request merged**